### PR TITLE
projects: detail view for HEP Valais

### PR DESCRIPTION
**Pull request by sebastiendeleze**
* Adds a custom detail view for HEP Valais projects.
* Closes rero/sonar#455.

Co-Authored-by: Sébastien Délèze <sebastien.deleze@rero.ch>

### CODE REVIEW

**Review by jma — APPROVED** [review]

File: projects/sonar/src/app/record/hepvs/project/detail/detail.component.html [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+  <div class="my-3" *ngIf="record.metadata.keywords">
+    <h5 class="d-inline mr-1" *ngFor="let keyword of record.metadata.keywords">
+      <span class="badge badge-secondary text-light font-weight-light">
+        <i class="fa fa-tag mx-1"></i> {{ keyword }}
+      </span>
+    </h5>
+  </div>
+
+  <p>{{ record.metadata.status }} ({{ record.metadata.startDate | dateTranslate: 'dd.MM.yyyy' }} - {{
+    record.metadata.endDate | dateTranslate: 'dd.MM.yyyy' }})</p>
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/hepvs/project/detail/detail.component.html:29]: Cosmitic.
`<p>{{ record.metadata.status }} ({{ record.metadata.startDate | dateTranslate: 'dd.MM.yyyy' }} -
{{ record.metadata.endDate | dateTranslate: 'dd.MM.yyyy' }})</p>`

File: projects/sonar/src/app/record/hepvs/project/detail/detail.component.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+      </dd>
+    </ng-container>
+
+    <!-- External partners -->
+    <ng-container *ngIf="record.metadata.externalPartners?.choice">
+      <dt class="col-sm-4 mt-2" translate>External partners</dt>
+      <dd class="col-sm-8 mt-2">
+        <ng-container *ngFor="let partner of record.metadata.externalPartners.list; let last = last">
+          {{ partner.searcherName }}<i class="text-muted ml-1" *ngIf="partner.institution">{{ partner.institution
+            }}</i>
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/hepvs/project/detail/detail.component.html:62]: Cosmitic

File: projects/sonar/src/app/record/hepvs/project/detail/detail.component.html [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+      <dd class="col-sm-8 mt-2">
+        <ng-container *ngIf="record.metadata.funding.funder.number">
+          <span class="badge badge-secondary text-light mr-1">
+            {{ record.metadata.funding.funder.number }}
+          </span>
+        </ng-container>
+        <ng-container *ngIf="record.metadata.funding.funder.name">
+          {{ record.metadata.funding.funder.name }}
+          <i class="text-muted ml-1" *ngIf="record.metadata.funding.funder.type !== 'Other (free field)'">{{
+            record.metadata.funding.funder.type }}</i>
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/hepvs/project/detail/detail.component.html:104]: cosmetic

File: projects/sonar/src/app/record/hepvs/project/detail/detail.component.html [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+    </ng-container>
+  </dl>
+
+  <ng-container *ngIf="user$ | async as user">
+    <ng-container *ngIf="record.metadata.documents && record.metadata.documents.length > 0 && user.is_moderator">
+      <h4 translate>Linked documents</h4>
+      <ul>
+        <li *ngFor="let document of record.metadata.documents">
+          <a [routerLink]="['/', 'records', 'documents', 'detail', document.pid]">{{ document.title[0].mainTitle |
+            languageValue }}</a>
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/hepvs/project/detail/detail.component.html:209]: cosmetic

**Review by Garfield-fr — APPROVED** [review]

**Pull request merged**